### DISCUSSION TITLE
Drug penetration into interstitial space

### DISCUSSION THREAD

**Discussion by thinguyen94** [Open-Systems-Pharmacology/Forum#1334] [Questions & Problems]
Hi PKSim users,

I'd like to ask about describing drug exposure in interstitial space in PKSim. I've optimized the plasma concentrations and then simulated concentrations in interstitial space of muscle tissue (left figure). By modifying endothelial permeability and surface area (as discussed previously in the forum), I could describe the elimination phase but couldn't describe the delay and the extent of tissue penetration (right figure).

I am looking at differential equations of passive diffusion (plasma to interstitial and interstitial to cell) in Mobi. However, I'm unsure how to modify them to prolong the drug absorption time into tissue but maintain the extent. Is there any time parameter in PKSim that I am unaware of?

Any comments would be highly appreciated. Thanks so much for your help.
![ISF modelling](https://user-images.githubusercontent.com/91042503/194827193-d1727401-3fc8-40fe-8303-857ccf2546b6.png)

My compound is a small molecule with:
- Molecular weight: 1449 g/mol
- logP: -3.1 or 2.2 (varies across studies and based on molecular structure)
- fu: 0.5-0.7
- pKa: 2.1, 7.7, 8.9

Best,
Thi

**Comment by wilbertdew** [comment]
Hi Thi,

My understanding is that the plasma interstitial permeability is set really high as an assumption. This parameter should mainly change the kinetics and not the extent (in equilibrium conditions). You can change this in MoBi but not in PK-Sim I believe? An interesting option here could be to model the molecule as a small molecule in the large molecule model, where the passive diffusion is set at a much lower permeability and the distribution kinetics are determined also by the two-pore model. I am not sure, but the default distribution kinetics might be a bit slower in this condition and you have more parameters that you can modify, such as the hydrodynamic radius. The last bit is a bit artificial and probably not what you want, but it might help. Correct me if I am wrong here @PavelBal

**Reply by thinguyen94** [reply]
Hi Wilberdew,

Thanks so much for your rapid response.

I found two relevant parameters and their formulas in MoBi:
1. PassiveDiffusionPl2Int = fu * P_endothelial * SA * (C_pls - C_int / K_int_pls)
2. PassiveDiffusionInt2Cell = K_water_int * PA_int_cell * C_int - K_water_cell * PA_cell_int * C_cell

But I don't know if modifying these formulas helps and if so how to do it.

The large molecule model sounds interesting - I wasn't aware of that. Will give it a try. Could you please explain the artificial bit? I don't know what it is. Thank you!

**Reply by wilbertdew** [reply]
Hi Thi,

Sorry for the delay. You can change the P_endothelial in the neighborhoods between plasma and interstitial in each tissue in a simulation. You can also find it in the parameter start values. I had a quick check, and the large molecule model has indeed slightly slower equilibration between plasma and interstitial, even without changing any parameters compared to the small molecule model for a small molecule. With the artificial bit, I meant that you could increase the hydrodynamic radius to delay the equilibration even if you know the hydrodynamic radius is not as big as you need to make it to decrease the rate sufficiently. Then you create an "artificial" situation just to fit the data, and you might later change the model differently if needed.

**Reply by thinguyen94** [reply]
Hi Wilbertdew,

Thanks so much for explaining. Yes I did change P_endothelial to get the above figure on the right. I was trying the large molecule model (attached). The equilibration was a bit slower (`~15 min), but not as much as I expected. The fit in plasma was worse than the previous. 

I optimised lymph flow, radius of pores and solute to simultanously fit plasma and tissue data. I also tried manually other vascular physiology in this large molecule model, but no improvement. Not sure how to describe better this 2-hour delay. Do you have any idea (of commonly optimised parameters)? Thanks so much.
![ảnh](https://user-images.githubusercontent.com/91042503/195516746-e9d6d230-c199-4e78-9293-ffc8fa6914e9.png)
